app/components/ui/atoms/NavigationDropdown.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { withTheme } from 'styled-components'
+import Select, { components } from 'react-select'
+
+import Icon from '../atoms/Icon'
+
+const DropdownArrow = props => (
+  <Icon iconName="ChevronDown" overrideName="dropdownArrow" {...props} />
+)
+
+class NavigationDropdown extends React.Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      value: this.props.defaultValue,
+    }
+
+    this.customReactSelectStyles = {
+      menuList: (base, state) => ({
+        ...base,
+        padding: 0,
+      }),
+      menu: (base, { placement }) => ({
+        ...base,
+        // remove default space between control & menu
+        marginBottom: 0,
+        // leave room for the bottom border of the Control component to be visible on focus
+        marginTop: this.props.theme.borderWidth,
+      }),
+      option: (base, { isSelected, isFocused }) => ({
+        ...base,
+        backgroundColor: this.props.theme.colorBackground,
+        color: this.props.theme.colorText,
```

Comment:
Sorry, should have made this more explicit in the top comment of the PR. Specs have changed - see [this comment from @chugginselifesciences](https://github.com/elifesciences/elife-xpub/issues/821#issuecomment-433848150)

Reply:
Spec has changed back - https://github.com/elifesciences/elife-xpub/issues/821#issuecomment-434665623

Doing this now :+1: 

Unfortunately, react-select doesn't easily allow you to pass in something like "initially selected item", so this isn't going to be quite as simple as it would be, if this component's state weren't controlled by the parent.

Hopefully won't take long